x-pack/legacy/plugins/siem/public/components/search_bar/index.tsx

```diff
@@ -41,7 +41,6 @@ import { timelineActions, hostsActions, networkActions } from '../../store/actio
 const {
   ui: { SearchBar },
   search,
-  timefilter,
```

Comment:
We would not need to do this change in SIEM codebase if we would re-export NP-`timefilter` from the shim.

Reply:
Worked with SIEM on this change. All good.